_docs/install/installing-drill-in-embedded-mode/011-running-drill-on-docker.md

```diff
@@ -1,4 +1,4 @@
---
+---
```

Comment:
Is it fixes this absent page?

Reply:
Oh geez thanks for the heads up about the link I completely forgot about that. I found that it actually does not fix it entirely as it goes to a page but its blank, and also the docker link is not found in the embedded mode section.

I'll look further into this.

Reply:
I had assumed that it would auto compile or some sort or perhaps I'm missing something? The header only had -- so replacing it with --- it now has the exact same format as all the other doc pages.

Reply:
It's odd... the path to it isnt even in the embedded section
![image](https://user-images.githubusercontent.com/71536798/108942950-ea32d700-7625-11eb-96e2-d261237e7d88.png)